src/transformers/models/llama/configuration_llama.py

```diff
@@ -83,14 +80,7 @@ class LlamaConfig(PretrainedConfig):
             Whether to tie weight embeddings
         rope_theta (`float`, *optional*, defaults to 10000.0):
             The base period of the RoPE embeddings.
-        rope_scaling (`Dict`, *optional*):
-            Dictionary containing the scaling configuration for the RoPE embeddings. Currently supports two scaling
-            strategies: linear and dynamic. Their scaling factor must be a float greater than 1. The expected format is
-            `{"type": strategy name, "factor": scaling factor}`. When using this flag, don't update
-            `max_position_embeddings` to the expected new maximum. See the following thread for more information on how
-            these scaling strategies behave:
-            https://www.reddit.com/r/LocalLLaMA/comments/14mrgpr/dynamically_scaled_rope_further_increases/. This is an
-            experimental feature, subject to breaking API changes in future versions.
+        {ROPE_CONFIG_DOCSTRING}
```

Comment:
I'd rather we left this abstraction for another time. At the moment, it just adds an extra step for anyone reading the code and wanting to find out the corresponding values. It's also a bit confusing wrt its names and the config signature, is this for `rope_scaling` or `rope_config`? 

Given how much happens to be repeated in our docstrings, it would be good to abstract out a lot of this, however the logic is inconsistent with other repeated fields we currently have. As we're moving to using the diff converter for adding models, I think this will become less of an issue.

Reply:
made it explicit :) 

Nevertheless, I would love to improve our copy for docstrings, the diff converter will only cover a few families of models! :) For instance, as I was working on cross-model propagation of these changes, I noticed that some docstrings items were missing